vignettes/modules.Rmd

```diff
@@ -30,19 +30,20 @@ library(mhealthtools)
 library(dplyr)
 ```
 
```

Comment:
> the front pocket of your pant while you walk

pants is always plural.

Reply:
> (also referred to as module, task or activity)

You didn't make the changes I requested in #144 about referring to the experimental design as an assay. Also module comes from the software engineering vernacular. It is a different concept from "experimental design".

Reply:
I learnt about modules as building blocks for entities. The entities can be programs, algorithms, or complex tasks, this is the actual definition.

For eg., look at this `https://en.wikipedia.org/wiki/Module_(mathematics)`, has been in use since long before the software engineering vernacular. 

BUT, if you think most of our readership would get confused about module, I can remove it. In my defense though, an experiment design is a complex task/activity and a module is its building block.

Reply:
Internally, we have only ever talked about modules in a software engineering context. I'm confused why it's mentioned as an alternative term for "Experimental Design" here. If you are using "module" here in its most context agnostic sense, it's like a cookbook saying "Please add the eggs (also known as food)".

Reply:
Oh.. alright if you put it that way. The point is to get the point across. I will remove module then.

Also that egg analogy is exactly not valid, as food forms a group in which eggs are one class of modules.